### PR TITLE
Update Developer Guide for Delete Command

### PR DESCRIPTION
**Pull request by parth-io**
In relation to #130

### CODE REVIEW

File: docs/diagrams/DeleteSequenceDiagram.puml [hunk 1]
```diff
@@ -3,47 +3,47 @@
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
-participant ":BookFaceParser" as BookFaceParser LOGIC_COLOR
+participant ":PrimaryParser" as PrimaryParser LOGIC_COLOR
 participant ":DeleteCommandParser" as DeleteCommandParser LOGIC_COLOR
-participant "d:DeleteCommand" as DeleteCommand LOGIC_COLOR
+participant ":DeleteUserCommand" as DeleteUserCommand LOGIC_COLOR
```
Comment (sprintaway) [inline docs/diagrams/DeleteSequenceDiagram.puml:8]: `DeleteUserCommand` is not used in this sequence diagram.

File: docs/diagrams/DeleteActivityDiagram.puml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+@startuml
+start
+:User enters a DeleteCommand;
+:DeleteCommandParser parses for valid format;
+'Since the beta syntax does not support placing the condition outside the
+'diamond we place it as the true branch instead.
+
+if (Delete command is valid?) then (yes)
```
Comment (sprintaway) [inline docs/diagrams/DeleteActivityDiagram.puml:8]: The recommended format in the text book is that the diamond node should be empty inside so can perhaps do 
```
if () then ([Delete command is valid])
...
else ([else])
```

instead

**Review by sprintaway — CHANGES_REQUESTED** [review]
LGTM

File: docs/diagrams/DeleteSequenceDiagram.puml (hunk 1 above)
Comment (parth-io) [reply docs/diagrams/DeleteSequenceDiagram.puml:8]: Yep, currently changing this to reflect the current procedure

File: docs/diagrams/DeleteActivityDiagram.puml (hunk 2 above)
Comment (parth-io) [reply docs/diagrams/DeleteActivityDiagram.puml:8]: Oh, thanks, I didn't know that

**Review by sprintaway — APPROVED** [review]

**Review by Edfernape — APPROVED** [review]
merging asap

**Pull request merged**